stable/nats/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nats
-version: 4.2.12
+version: 4.2.13
```

As it is a new feature we usually bump the chart version in a minor.
```suggestion
version: 4.3.0
```